### ISSUE TITLE
Undefined class constant 'EVENT_REGISTER_GQL_SCHEMA_COMPONENTS'

### ISSUE THREAD

**Issue by russback** [nystudio107/craft-retour#198]
### Describe the bug
Updating from retour 3.1.41 => 3.1.58 on Craft v 3.4.30.

### To reproduce
1. Run ./craft update retour 
3. See the following error:

```
Applying new migrations ... error: The command "'/{MYPATH}/craft' 'migrate/all' '--no-backup' '--no-content'" failed.

Exit Code: 1(General error)

Working directory: /{MYPATH}

Output:
================


Error Output:
================
Exception 'Error' with message 'Undefined class constant 'EVENT_REGISTER_GQL_SCHEMA_COMPONENTS''

in /{MYPATH}/vendor/nystudio107/craft-retour/src/Retour.php:460
```

### Expected behaviour
This site has not been upgraded beyond Craft 3.4 but assumed a patch release of Retour would still work.

### Versions
- Plugin version: 3.1.41 => 3.1.58
- Craft version: 3.4.30
- PHP version: 7.4

**Comment by khalwat** [comment]
Will fix this today, sorry about that

**Comment by khalwat** [comment]
Fixed in https://github.com/nystudio107/craft-retour/commit/11efd8e3539f887f5bfe688390d6dcd8aca52055

**Comment by khalwat** [comment]
Released as Retour 3.1.59 -> https://github.com/nystudio107/craft-retour/releases/tag/3.1.59
